src/com/palmergames/bukkit/towny/command/BaseCommand.java

```diff
@@ -360,4 +360,15 @@ public static void checkPermOrThrow(Permissible permissible, String node) throws
 	public static void checkPermOrThrowWithMessage(Permissible permissible, String node, Translatable errormsg) throws NoPermissionException {
 		TownyUniverse.getInstance().getPermissionSource().testPermissionOrThrow(permissible, node, errormsg);
 	}
+
+	public static List<String> getTrustedTownsFromResident(Player player){
+		Resident res = TownyUniverse.getInstance().getResident(player.getUniqueId());
+
+		if (res != null && res.hasTown()) {
+			return res.getTownOrNull().getTrustedTowns().stream().map(Town ->Town.getName())
+				.collect(Collectors.toList());

```

Comment:
```suggestion
			return res.getTownOrNull().getTrustedTowns().stream().map(TownyObject::getName)
				.collect(Collectors.toList());

```

Reply:
Have implemented change as suggested in commit.